report-viewer/src/router/index.ts

```diff
@@ -0,0 +1,33 @@
+import {createRouter, createWebHistory, RouteRecordRaw} from "vue-router";
+import Overview from "@/views/OverviewView.vue";
+import ComparisonView from "@/views/ComparisonView.vue"
+import FileUpload from "@/views/FileUpload.vue"
+
+const routes: Array<RouteRecordRaw> = [
+  {
+    path: "/",
+    name: "FileUpload",
+    component: FileUpload
+  },
+  {
+    path: "/overview",
+    name: "OverviewView",
+    component: Overview,
+  },
+  {
+    path: "/comparison",
+    name: "ComparisonView",
+    component: ComparisonView,
+    props: route => ({
+      id1: route.query.id1,
```

Comment:
It's known as bad practice to name variables nearly identical, i.e. `id1` and `id2`. A spelling mistake is made quickly and then hard to find. Better use something like `firstId`.